lib/rspec/support/object_formatter.rb

```diff
@@ -2,10 +2,38 @@ module RSpec
   module Support
     # Provide additional output details beyond what `inspect` provides when
     # printing Time, DateTime, or BigDecimal
```

Comment:
I also removed the individual ```# @private``` comments, please let me know if this wasn't the correct thing to do